docs/spec.md

```diff
@@ -4154,7 +4174,7 @@ Afterwards, within each `process_group`:
 
 | Label | Name                    | Type                                         | Constraints            |
 |-------|-------------------------|----------------------------------------------|------------------------|
-| (I1)  | `operand`               | tensor                                       | (C1), (C2), (C7), (C8) |
+| (I1)  | `operand`               | tensor or quantized tensor                   | (C1), (C2), (C7), (C8) |
```

Comment:
Similar to line 792, should "quantized tensor" be per-tensor quantized tensor or per-axis constraints be added?

Reply:
Thanks for the catch! It was unintended and now fixed.